context/app/static/js/components/workspaces/AdvancedConfigOptions/AdvancedConfigOptions.tsx

```diff
@@ -145,26 +138,28 @@ function AdvancedConfigOptions<FormType extends FieldValues>({
   return (
     <StyledAccordion>
       <AccordionSummary expandIcon={<ArrowDropDownRounded color="primary" />}>
-        <StyledHeader>Advanced Configurations (Optional)</StyledHeader>
+        <Typography variant="button" fontSize="1rem">
+          Advanced Configurations (Optional)
+        </Typography>
       </AccordionSummary>
       <AccordionDetails>
         <Stack>
-          <Typography>{description}</Typography>
-          <SpacedSectionButtonRow
-            buttons={
-              <Stack direction="row" gap={1} marginTop={1}>
-                <Button type="button" variant="contained" disabled={isDefault} onClick={handleRestoreDefaults}>
-                  Restore Defaults
-                </Button>
-              </Stack>
-            }
-          />
+          <Typography marginBottom={1}>{description}</Typography>
+          <Button
+            type="button"
+            variant="contained"
+            sx={{ alignSelf: 'flex-end' }}
+            disabled={isDefault}
+            onClick={handleRestoreDefaults}
+          >
+            Restore Defaults
+          </Button>
           {configSliderOptions.map((props) => (
             <ConfigSlider key={props.id} field={field} {...props} />
           ))}
-          <StyledSubtitle>Enable GPU</StyledSubtitle>
+          <Typography variant="button">Enable GPU</Typography>
           <Stack direction="row" component="label" alignItems="center">
-            <StyledSwitchLabel>Disabled</StyledSwitchLabel>
+            <Typography fontSize=".75rem">Disabled</Typography>
```

Comment:
Is there not a typography variant in the theme that matches the style?

Reply:
Good catch, the `caption` variant works in this case.